src/index.js

```diff
@@ -87,13 +88,21 @@ class JanusAdapter {
 
   connect() {
     debug(`connecting to ${this.serverUrl}`);
-    this.ws = new WebSocket(this.serverUrl, "janus-protocol");
-    this.session = new mj.JanusSession(this.ws.send.bind(this.ws));
-    this.ws.addEventListener("open", this.onWebsocketOpen);
-    this.ws.addEventListener("message", this.onWebsocketMessage);
+    this.ws = new WebSocketClient(this.serverUrl, "janus-protocol", {
+      strategy: 'fibonacci',
+      randomisationFactor: 0, // defaults to 0, must be between 0 and 1
+      initialDelay: 1,       // defaults to 100 ms
+      maxDelay: 300           // defaults to 10000 ms
+    });
```

Comment:
Where did you learn about these parameters? If I go look at the source in https://github.com/gamestdio/websocket/blob/master/src/backoff.js I see only `initialDelay` -- I don't see `randomisationFactor` or `maxDelay`.

Maybe this should just be
```Javascript
this.ws = new WebSocketClient(this.serverUrl, "janus-protocol", {
  backoff: 'fibonacci',
  initialDelay: '1'
});
```
?

Reply:
yeah looks like it was removed in this commit https://github.com/gamestdio/websocket/commit/717a90b13c86bea55e7e315829e05c0dd5416934

i will update the PR